alibi_detect/saving/tests/test_saving.py

```diff
@@ -1132,14 +1130,40 @@ def test_save_preprocess(data, preprocess_fn, tmp_path, backend):
         assert isinstance(preprocess_fn_load.keywords['model'], nn.Module)
 
 
+@parametrize('preprocess_fn', [preprocess_simple, preprocess_simple_with_kwargs])
+def test_save_preprocess_custom(preprocess_fn, tmp_path):
```

Comment:
New test function to test save/load of:

1. `preprocess_fn` where it is a Python func with no kwargs.
2. `preprocess_fn` where it is a `partial` with kwargs.